### PR TITLE
Styling updates.

### PR DESCRIPTION
**Pull request by dzearing**
This includes the changes to the styling package from the styling branch.

* All exported const collections now use TitleCasing for exported consts.
* The theme `colors` property has been renamed to `palette`.
* The `css` helper (which is confusingly named, considering the redundant utilities package `css` helper) has been renamed to `mergeStyles`.
* Various typings have been updated.

### CODE REVIEW

File: packages/styling/src/examples/ColorPage/ColorPage.styles.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  swatchText?: IStyle;
+  example?: IStyle;
+}
+
+export function getStyles(theme: ITheme = getTheme()): IColorPageStyles {
+  return {
+    swatch: mergeStyles({
+      display: 'inline-block',
+      verticalAlign: 'middle',
+      width: 20,
```
Comment (christiango) [inline packages/styling/src/examples/ColorPage/ColorPage.styles.ts]: > 20 [](start = 12, length = 3)

Any reason these don't have units?

File: packages/styling/src/styles/DefaultFontStyles.ts [hunk 2]
```diff
@@ -1,66 +1,55 @@
...
-const SMALL_SIZE: string = '12px';
-const SMALL_PLUS_SIZE: string = '13px';
-const MEDIUM_SIZE: string = '14px';
-const MEDIUM_PLUS_SIZE: string = '15px';
-const LARGE_SIZE: string = '17px';
-const X_LARGE_SIZE: string = '21px';
-const XX_LARGE_SIZE: string = '28px';
-const SUPER_SIZE: string = '42px';
-const MEGA_SIZE: string = '72px';
+export namespace FontSizes {
```
Comment (christiango) [inline packages/styling/src/styles/DefaultFontStyles.ts]: What's the reason behind the namespace export instead just exporting an object with these properties?

File: packages/styling/src/examples/ColorPage/ColorPage.styles.ts (hunk 1 above)
Comment (dzearing) [inline packages/styling/src/examples/ColorPage/ColorPage.styles.ts]: less bytes. Default unit for numbers is px.

File: packages/styling/src/styles/DefaultFontStyles.ts (hunk 2 above)
Comment (cschleiden) [inline packages/styling/src/styles/DefaultFontStyles.ts]: For an object you'd have to do something like this? 

```
const x = {
    foo: 42,
    bar: "test"
};
export const a: Readonly<typeof x> = x;

////

export namespace b {
    export const foo = 42;
    export const bar = "test";
}
```

or is there a better way?

Comment (dzearing) [inline packages/styling/src/styles/DefaultFontStyles.ts]: Reverting.

Comment (dzearing) [inline packages/styling/src/styles/DefaultFontStyles.ts]:
```ts
export const IconCodes = {
  code1: 'foo',
  code2: 'bar',
  etc
};
```

This is better imo.

Comment (dzearing) [inline packages/styling/src/styles/DefaultFontStyles.ts]: Another question; should the values in this map be TitleCased or camelCased?

Comment (dzearing) [inline packages/styling/src/styles/DefaultFontStyles.ts]: For existing ms-Icon classnames, the icon names are TitleCased.

For const exports, we should follow the airbnb guideline of TitleCasing them.

If I had kept the namespace, we'd be exporting TitleCased consts.

Comment (cschleiden) [inline packages/styling/src/styles/DefaultFontStyles.ts]: @dzearing That doesn't prevent:

```
IconCodes.code1 = "newvalue";
```

it only prevents

```
IconCodes = {};
```

Comment (cschleiden) [inline packages/styling/src/styles/DefaultFontStyles.ts]: You could also do 
```
const makeReadonly = <T>(value): Readonly<T> => value; 

export const IconCodes = makeReadonly({
  foo: "bar",
  key: "value"
});
```

**Pull request merged**